docs/cluster-management/developing-locally.md

```diff
@@ -248,39 +250,38 @@ git clone git@github.com:screwdriver-cd/queue-worker.git
     # Externally routable URL for the Artifact Store
     store: http://$YOUR_IP:9002
     
- redis:
-    # Host of redis cluster
-    host: 127.0.0.1
-    # Password to connect to redis cluster, if set in first step
-    password: 'c1cd-$cr3wdriver-cd'
-    # Port of redis cluster
-    port: 6379
-    # Prefix for the queue name if needed
-    # prefix: 'beta-'
-    # Flag to enable client for TLS-based communication
-    tls: false
+ queue:
+    # Configuration of the redis instance containing resque
+    redisConnection:
+        host: "127.0.0.1"
+        port: 6379
+        options:
+            # Set password if set in previous step
+            password: 'c1cd-$cr3wdriver-cd'
```

Comment:
maybe just put something that looks less like a real password or just leave it blank

Reply:
also what does this comment mean?
>Set password if set in previous step